src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -14038,15 +14025,13 @@ private new ResetPoint GetResetPoint()
             return new ResetPoint(
                 base.GetResetPoint(),
                 _termState,
-                _isInTry,
```

Comment:
removing context we used to have save/restore (but didn't store within nodes (like we do with our other contextual flags)).